index.rst

```diff
@@ -0,0 +1,23 @@
+.. MbedTLS documentation master file, created by
+   sphinx-quickstart on Wed Jul 21 18:29:51 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to MbedTLS's documentation!
+===================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   reviews/review_guidelines.md
+   architecture/proposed/long-term-plans.md
+   security-advisories/security-advisories.md
+   maintainers.md
```

Comment:
Was this automatically added by `sphinx-quickstart`? It wasn't when I tried it with Sphinx 1.8.5. If not, please make that in a separate commit.

Reply:
Ack - this was added by me, so I'll do a separate commit.